docs/en/migration.md

```diff
@@ -442,7 +466,7 @@ init_cfg=dict(
     checkpoint='torchvision://resnet50')
 ```
 
-In addition to these commonly used backbones, you can easily use backbones from repositories in the OpenMMLab ecosystem such as MMClassification, which all share the same config system and provide pre-trained weights.
```

Comment:
`OpenMMLab ecosystem ` may be confused with https://github.com/open-mmlab/ecosystem